deploy/operator/operator.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nmstate-operator
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: nmstate-operator
+  template:
+    metadata:
+      labels:
+        app: kubernetes-nmstate
+        name: nmstate-operator
+    spec:
+      serviceAccountName: nmstate-operator
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: nmstate-operator
+          args:
+          - --v=production
+          # Replace this with the built image name
+          image: {{ .NMStateHandlerImage }}
+          imagePullPolicy: {{ .ImagePullPolicy }}
+          command:
+          - kubernetes-nmstate
+          env:
+            - name: WATCH_NAMESPACE
+              value: ""
+            - name: OPERATOR_NAME
+              value: "kubernetes-nmstate-operator"
+            - name: ENABLE_PROFILER
+              value: "False"
+            - name: PROFILER_PORT
+              value: "6060"
+            - name: RUN_OPERATOR
+              value: ""
```

Comment:
I guess "True" is more suitable in that case, right?

Reply:
The usual way of do this flag things at pods is just adding the var without value, so we don't care about the value, at golang we can check that var is set or not and that's enough, no need to parse a string value (numerical boolean ? etc...).